### PR TITLE
made img tag self closing in index.html, home-index.html and kitchensink.html

### LINKED ISSUE #297: [Bug] Img tags not self closing across various contexts
## Description

With reference to #283 , this focuses on the presence of non closing `img` tags in the following contexts;

- `index.html` file in the docs folder
- `home-index.html` file in the specimen/contexts folder
- `kithensink.html` in the specimen/tests folder 

## Reproduction

1. Fork and clone the repo to your local machine
2. Go to the search pane of your code editor ( VS code in my case)
3. use the following regex to search for non-closing `img` tags; `]*?(?<!/)>`
4. Observe the non closing `img` tags

## Expectation

You would observe the non closing `img` tags.

## Screenshots

image

## Environment

 - Device: (laptop)
 - OS: (windows)
 - Browser: (chrome)

## Resolution

- [x] I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by shivam8112005**
## Fixes 

- Fixes #297  by @egbadon-victor 

## Description
img tag is not self closing in : 
1. index.html file in the docs folder
2. home-index.html file in the specimen/contexts folder
3. kithensink.html in the specimen/tests folder

## Technical details
some places in the project closed img tag() is used but somewhere it was not closed

## Tests
1. Fork and clone the repo to your local machine
2. Go to the search pane of your code editor ( VS code in my case)
3. use the following regex to search for non-closing img tags; ]*?(?<!/)>
4. Observe the closed img tags

## Screenshots
Screenshot 2024-10-29 000433

## Checklist

- [X] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [X] My pull request targets the *default* branch of the repository (`main` or `master`).
- [X] My commit messages follow [best practices][best_practices].
- [X] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [X] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -426,7 +426,7 @@ <h2>Images Attribution</h2>
       <li>
         <article>
           <figure>
-            <img src="PATH/TO/image.png">
+            <img src="PATH/TO/image.png"/>
```
Comment (possumbilities) [inline docs/index.html]: should be a space preceding the `/`

Comment (shivam8112005) [reply docs/index.html]: ohk wait

**Review by possumbilities — APPROVED** [review]

**Comment by possumbilities** [comment]
@shivam8112005 If you comment on the original issue I can credit you

**Comment by shivam8112005** [comment]
@possumbilities credit by what?

**Comment by possumbilities** [comment]
@shivam8112005 so I can assign the issue #297 you just closed to you.

**Comment by shivam8112005** [comment]
@possumbilities ohk i will mark it from now on

**Comment by possumbilities** [comment]
@shivam8112005 I think perhaps we're miscommunicating. You need to go to #297 and leave a comment on the issue, afterwards it will let me Assign you to it, so that the Issue can be fully resolved.

**Comment by shivam8112005** [comment]
wait im commenting

**Comment by shivam8112005** [comment]
@possumbilities  i have commented on #297 check that out

**Pull request merged**